wifi_logger.c

```diff
@@ -237,29 +264,53 @@ void wifi_logger()
 #ifdef CONFIG_TRANSPORT_PROTOCOL_TCP
 void wifi_logger()
```

Comment:
1) Something seems wrong with whitespaces, can you use "TAB" for 4 whitespaces everywhere.
2) You need to make sure we are freeing variable handle whenever breaking out

Reply:
1. it shows correctly indented on my IDE. 
2. it is already done in tcp_network_manager before returning

Reply:
1) There's something wrong with github, this happens too often.
2) Ohk, just wanted to make sure, as malloc in a while loop appears to be dangerous

Reply:
1. i changed my indentation to 4 spaces instead of a tab. it should be fine now
2. very true, that's why i took the log_message variable outside. Constantly allocating and freeing memory was waste of time, i thought

Reply:
Cool, I'll merge the pull request